linode/manager
# M3-5700: Linode Create Replace Checkout Sidebar with Summary Paper

## Description

- Removes the Checkout sidebar from the Linode Create page
- Adds a new component called CheckoutSummary that goes at the bottom of the create page

## How to test

- Check against the internal mock-ups
- Test many different Linode configurations and check the Summary at the bottom for style and functionality
